ports/stm32/sdcard.c

```diff
@@ -187,7 +177,7 @@ STATIC void sdmmc_msp_init(void) {
 
     #if defined(STM32H7)
     // Reset SDMMC
-    #if defined(MICROPY_HW_SDMMC2_CK)
+    #if MICROPY_HW_SDCARD_SDMMC == 2
     __HAL_RCC_SDMMC2_FORCE_RESET();
     __HAL_RCC_SDMMC2_RELEASE_RESET();
```

Comment:
```suggestion
    SDMMC_FORCE_RESET();
    SDMMC_RELEASE_RESET();
```
Need to define macros for these as well... I missed that in my PR.